client/clip_client/client.py

```diff
@@ -334,6 +341,7 @@ def encode(self, content, **kwargs):
                 on_error=on_error,
                 on_always=partial(self._update_pbar, func=on_always),
                 parameters=parameters,
+                prefetch=prefetch,
```

Comment:
You can just inline the code like: `prefecth=kwargs.get('prefetch', 1000)`.  You can reduce the number of lines if you do this to other params also.